docs/source/user-guide/concepts-readings-events.md

```diff
@@ -36,6 +37,22 @@
 
 This is a list of DataFusion related blog posts, articles, and other resources. Please open a PR to add any new resources you create or find
 
+- **2024-11-22** [Blog: Apache Datafusion Comet and the story of my first contribution to it](https://semyonsinchenko.github.io/ssinchenko/post/comet-first-contribution/)
+
+- **2024-11-21** [Blog: DataFusion is featured as one of the coolest 10 open source software tools by CRN](https://www.crn.com/news/software/2024/the-10-coolest-open-source-software-tools-of-2024?page=3)
+
+- **2024-11-20** [Apache DataFusion Comet 0.4.0 Release](https://datafusion.apache.org/blog/2024/11/20/datafusion-comet-0.4.0/)
```

Comment:
Should we include all releases or just provide a parent link to all releases notes? 🤔

Reply:
This one was a blog post so I think it deserves its own line on this page

I would be happy to remove these announcements if you prefer, but I think it is kind of nice to show momentum